packages/src/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Internal dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+const {
+    registerBlockType,
+    unstable__bootstrapServerSideBlockDefinitions, // eslint-disable-line camelcase
+} = wp.blocks;
+
+/**
+ * Function to register core blocks provided by the block editor.
+ */
+
+const registerGlutenblocks = () => {
+    [
+    ].forEach((block) => {
+        if (!block) {
+            return;
+        }
+        const { metadata, settings, name } = block;
+        if (metadata) {
+            unstable__bootstrapServerSideBlockDefinitions({ [name]: metadata }); // eslint-disable-line camelcase
+        }
+        registerBlockType(name, settings);
+    });
+};
+
+registerGlutenblocks();
```

Comment:
Why define a `const` instead of calling the method directly?

```js
(function registerGlutenblocks() {
    //
})();
```

Reply:
The suggestion was implemented.